client/src/components/Hero.js

```diff
@@ -15,19 +15,15 @@ function Hero() {
           <Image  src="https://i.ibb.co/j6f02Js/kunji-1.jpg" alt="Right Column Image" fluid className={styles.heroImages} />
         </Col>
       </Row >
-    </Container>
-    <Container>
       <Row className={styles.rowMain}>
         <Col md={6}>
-          <Image src="https://i.ibb.co/HgTHb2j/kunji-2.jpg" alt="Right Column Image" fluid className={styles.heroImages} />
+          <Image src="https://i.ibb.co/HgTHb2j/kunji-2.jpg" alt="Left Column Image" fluid className={styles.heroImages} />
```

Comment:
The alt is a textual alternative of the image - I.e. brief description of what is in the image

https://www.w3schools.com/TAGS/att_img_alt.asp

Reply:
Done.